### PR TITLE
add links to listings of non-compliant packages

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/25007

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
Proposal for some rearrangements with "Systems" menu and "Systems Details" tab.  "Systems Details" tab isn't a normal submenu of "Systems" but a hierarchy of its own.  Let's add it at the same level until we will find a even better solution.  Adding screenshots now...
Systems: Screenshot from 2024-08-14 11-16-41
System Details: Screenshot from 2024-08-14 11-17-32

File: modules/reference/pages/systems/system-details/sd-details.adoc [hunk 1]
```diff
@@ -5,9 +5,24 @@ The [guimenu]``System Details`` section allows you to manage the details for a c
...
 The [guimenu]``System Details`` section is divided into tabs and subtabs.
-Each tab and subtab has its own section in this document.
+Each tab and some subtabs have own sections in this document.
 
 The menu:Systems[Details > Overview] subtab is displayed by default.
-Which tabs are available is dependent on the system type of the client you have selected.
+Which tabs are available is dependent on the [systemitem]``system type`` of the client you have selected.
+
+////
+// FIXME Do we want to display the list here:
```
Comment (Bischoff) [inline modules/reference/pages/systems/system-details/sd-details.adoc:14]: hmmm, could be a good idea, but you're the master here...

File: modules/reference/pages/systems/systems-list.adoc [hunk 2]
```diff
@@ -58,7 +58,8 @@ The menu:Systems[System List > Requiring Reboot] section lists every client that
 
 The menu:Systems[System List > Non Compliant] section lists every client that has non-compliant packages installed.
 Packages are considered non-compliant if they have been installed from a source other than {productname}.
-In the list, the [guimenu]``Packages`` column shows the number of non-compliant packages on the client.
+In the list, the [guimenu]``Extra Packages`` column shows the number of non-compliant packages on the client.
+These numbers are links to listings of those non-compliant packages per system.
```
Comment (Bischoff) [inline modules/reference/pages/systems/systems-list.adoc:62]: Cool.

Beware: if the number is zero, there's no link, just the number

add ", unless there are no such packages." ?

File: modules/reference/pages/systems/systems-menu.adoc [hunk 3]
```diff
@@ -3,4 +3,46 @@
...
+* [guimenu]``Proxy Configuration``  - xref:reference:systems/proxy-configuration.adoc[]
+* [guimenu]``Advanced Search``	    - xref:reference:systems/advanced-search.adoc[]
+* [guimenu]``Activation Keys``	    - xref:reference:systems/activation-keys.adoc[]
+* [guimenu]``Stored Profiles``	    - xref:reference:systems/stored-profiles.adoc[]
+* [guimenu]``Custom System Info``   - xref:reference:systems/custom-system-info.adoc[]
+* [guimenu]``Autoinstallation``	    - xref:reference:systems/autoinst-menu.adoc[]
+* [guimenu]``Virtual Host Managers` - xref:reference:systems/virtual-host-managers.adoc[]`
+
+
+Click the name of a client to go to that client system's the [guimenu]``Details`` overview page and all its tab and subtabs.
```
Comment (Bischoff) [inline modules/reference/pages/systems/systems-menu.adoc:38]: cool

File: modules/reference/pages/systems/systems-overview.adoc [hunk 4]
```diff
@@ -1,19 +1,22 @@
 [[ref-systems-overview]]
-= Systems Overview
+= Systems Menu Entries (Overview)
```
Comment (Bischoff) [inline modules/reference/pages/systems/systems-overview.adoc:2]: so true...

**Review by Bischoff — CHANGES_REQUESTED** [review]
lgtm, see nitpicks

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+- Improved documentation arund non-compliant packages (also known as extra
+  packages) in Reference Guide.
+- Restructured documenation of Systems menu and system details tab in
```
Comment (keichwa) [inline CHANGELOG.md:3]:
```suggestion
- Restructured documentaion of Systems menu and system details tab in
```

File: modules/reference/pages/systems/systems-list.adoc (hunk 2 above)
Comment (keichwa) [inline modules/reference/pages/systems/systems-list.adoc:62]: ok, but I try to think in a positive way:
```suggestion
If there is at least one non-compliant package, you can click the number to display listings of those non-compliant packages per system.
```

File: modules/reference/pages/systems/systems-menu.adoc (hunk 3 above)
Comment (keichwa) [inline modules/reference/pages/systems/systems-menu.adoc:38]:
```suggestion
Click the name of a client to go to that client system's [guimenu]``Details`` overview page and all its tabs and subtabs.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+- Improved documentation arund non-compliant packages (also known as extra
+  packages) in Reference Guide.
+- Restructured documentaion of Systems menu and system details tab in
```
Comment (keichwa) [inline CHANGELOG.md:3]:
```suggestion
- Restructured documentation of Systems menu and system details tab in
```

**Review by 0rnela — APPROVED** [review]

**Pull request merged**